From 794671994d49f6a00c0d4a4f6f2d9df805e01fca Mon Sep 17 00:00:00 2001
From: Leandro Ribeiro <leandrohr@riseup.net>
Date: Tue, 29 Oct 2019 22:12:11 -0300
Subject: drm/doc: Update IGT documentation
Git-commit: 8a537de0f3d8b655cb901c948ed863bf0b23277b
Patch-mainline: v5.6-rc1
References: jsc#SLE-12680, jsc#SLE-12880, jsc#SLE-12882, jsc#SLE-12883, jsc#SLE-13496, jsc#SLE-15322

The IGT documentation in this page is telling us to build it using
make. According to commit 67993c1 ("automake: Point builders at
meson") from the IGT project, this is deprecated and IGT should be
built with meson. Instead of having a documentation for IGT in this
page, point to their GitLab README, which should always be up to
date.

Signed-off-by: Leandro Ribeiro <leandrohr@riseup.net>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Link: https://patchwork.freedesktop.org/patch/msgid/20191030011211.47942-1-leandrohr@riseup.net
Signed-off-by: Patrik Jakobsson <pjakobsson@suse.de>
---
 Documentation/gpu/drm-uapi.rst | 32 ++------------------------------
 1 file changed, 2 insertions(+), 30 deletions(-)

diff --git a/Documentation/gpu/drm-uapi.rst b/Documentation/gpu/drm-uapi.rst
index 94f90521f58c..996f7354e05c 100644
--- a/Documentation/gpu/drm-uapi.rst
+++ b/Documentation/gpu/drm-uapi.rst
@@ -254,36 +254,8 @@ Validating changes with IGT
 There's a collection of tests that aims to cover the whole functionality of
 DRM drivers and that can be used to check that changes to DRM drivers or the
 core don't regress existing functionality. This test suite is called IGT and
-its code can be found in https://cgit.freedesktop.org/drm/igt-gpu-tools/.
-
-To build IGT, start by installing its build dependencies. In Debian-based
-systems::
-
-	# apt-get build-dep intel-gpu-tools
-
-And in Fedora-based systems::
-
-	# dnf builddep intel-gpu-tools
-
-Then clone the repository::
-
-	$ git clone git://anongit.freedesktop.org/drm/igt-gpu-tools
-
-Configure the build system and start the build::
-
-	$ cd igt-gpu-tools && ./autogen.sh && make -j6
-
-Download the piglit dependency::
-
-	$ ./scripts/run-tests.sh -d
-
-And run the tests::
-
-	$ ./scripts/run-tests.sh -t kms -t core -s
-
-run-tests.sh is a wrapper around piglit that will execute the tests matching
-the -t options. A report in HTML format will be available in
-./results/html/index.html. Results can be compared with piglit.
+its code and instructions to build and run can be found in
+https://gitlab.freedesktop.org/drm/igt-gpu-tools/.
 
 Display CRC Support
 -------------------
-- 
2.28.0

